base/dates/periods.jl

```diff
@@ -377,99 +377,99 @@ Base.isequal(x::Period, y::CompoundPeriod) = isequal(y, x)
 Base.isequal(x::CompoundPeriod, y::CompoundPeriod) = x.periods == y.periods
 
 # Capture TimeType+-Period methods
-(+)(a::TimeType,b::Period,c::Period) = (+)(a,b+c)
-(-)(a::TimeType,b::Period,c::Period) = (-)(a,b-c)
-(+)(a::TimeType,b::Period,c::Period,d::Period...) = (+)((+)(a,b+c),d...)
-(-)(a::TimeType,b::Period,c::Period,d::Period...) = (-)((-)(a,b-c),d...)
+(+)(a::TimeType, b::Period, c::Period) = (+)(a, b + c)
+(-)(a::TimeType, b::Period, c::Period) = (-)(a, b - c)
+(+)(a::TimeType, b::Period, c::Period, d::Period...) = (+)((+)(a, b + c), d...)
+(-)(a::TimeType, b::Period, c::Period, d::Period...) = (-)((-)(a, b - c), d...)
 
-function (+)(x::TimeType,y::CompoundPeriod)
+function (+)(x::TimeType, y::CompoundPeriod)
     for p in y.periods
         x += p
     end
     return x
 end
-(+)(x::CompoundPeriod,y::TimeType) = y + x
+(+)(x::CompoundPeriod, y::TimeType) = y + x
 
-function (-)(x::TimeType,y::CompoundPeriod)
+function (-)(x::TimeType, y::CompoundPeriod)
     for p in y.periods
         x -= p
     end
     return x
 end
-(-)(x::CompoundPeriod,y::TimeType) = y - x
+(-)(x::CompoundPeriod, y::TimeType) = y - x
```

Comment:
another instance of commutative subtraction that should be deprecated

(looks like @GordStephen may have been the author of a few of these)